tests/testPsfDetermination.py

```diff
@@ -466,7 +468,7 @@ def testRejectBlends(self):
 
         with self.assertRaises(RuntimeError) as cm:
             psfDeterminer.determinePsf(self.exposure, candidates, metadata)
-        self.assertEqual(cm.exception.message, "All PSF candidates removed as blends")
+        self.assertEqual(cm.exception.args[0], "All PSF candidates removed as blends")
```

Comment:
`str(cm.exception)` might be better. `.message` is now what happens when you stringify the exception.